test_runner/fixtures/neon_fixtures.py

```diff
@@ -3497,6 +3521,9 @@ def start(
             basebackup_request_tries=basebackup_request_tries,
         )
         self._running.release(1)
+        self.log_config_value("shared_buffers")
+        self.log_config_value("neon.max_file_cache_size")
+        self.log_config_value("neon.file_cache_size_limit")
```

Comment:
I think that's a bit too much logging by default

Reply:
This allows us to be sure that LFC is in use or not.